recipes/arraylake/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "arraylake" %}
+{% set version = "0.7.2a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arraylake-{{ version }}.tar.gz
+  sha256: d5674aac64c3aac2c02b0ed38d250c0c9d30c37c8428f359d39cb2e9269370a5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - arraylake = arraylake.cli.main:app
+    - al = arraylake.cli.main:app
+noarch_platforms:
+  - linux_64
+  - win_64
```

Comment:
Think this will need to be added to `conda-forge.yml` once the feedstock is created (for [example]( https://github.com/conda-forge/jupyterhub-feedstock/blob/b849fbd2ae0675f6df8e9da1fb26c24f99c43cbb/conda-forge.yml#L9-L11 ))

Would drop it from `meta.yaml`

```suggestion
```